src/_sass/_colours.scss

```diff
@@ -3,6 +3,7 @@ $cream: #f3f1e9;
 $ivory: #f9f8f5;
 
 $yellow-light: #fcd672;
+$blue-mid: #009abf;
```

Comment:
This is one of the brand colours (marked below), but are you thinking it would be better to use one of the non-mid/non-light hues?

<img width="1004" alt="Screenshot 2022-11-24 at 11 13 53" src="https://user-images.githubusercontent.com/40244233/203770759-b68ee7b9-adfb-4f6e-bac3-c8cddd9c04a1.png">